HoloPipelines/pipelines/components/compDicom2numpy.py

```diff
@@ -79,10 +79,19 @@ def resample(dataPath, new_spacing=[1, 1, 1]):
     return image, new_spacing
 
 
+def reorientateNumpyList(numpyList):
+    # transpose numpy i.e. (z, y, x) ---> (x, y, z)
+    numpyList = numpyList.transpose(2, 1, 0)
+    numpyList = np.flip(numpyList, 0)
+    numpyList = np.flip(numpyList, 1)
+    return numpyList
+
+
 def main(dicomPath):
     print("dicom2numpy: resampling dicom...")
```

Comment:
change all print to logging.info or something.

Reply:
update print too logging.info and logging.warning